src/groups/bmq/bmqt/bmqt_subscription.h

```diff
@@ -203,9 +203,19 @@ class Subscription {
 
     /// Create a new Subscription by copying values from the specified
     /// `other`.
-    Subscription(const Subscription& other);
+    explicit Subscription(const Subscription& other);
 
     // MANIPULATORS
+    /// Assign to this object the value of the specified `rhs` object.
+    Subscription& operator=(const Subscription& rhs);
+
+#if defined(BSLS_COMPILERFEATURES_SUPPORT_RVALUE_REFERENCES) &&               \
+    defined(BSLS_COMPILERFEATURES_SUPPORT_NOEXCEPT)
+    /// Assign to this object the value of the specified `rhs` object.
+    /// After performing this action, the `rhs` object will be left in a
+    /// valid, but unspecified state.
+    Subscription& operator=(Subscription&& rhs);
+#endif
```

Comment:
We should probably use `bslmf::MovableRef` instead of preprocessor guards here

Reply:
On the other hand, what do you think about removing explicit copy constructor and assignment at all? It should deal with the warnings, and also we re-implemented the default one anyway. We don't use allocators for these classes, so we don't need to implement allocator-aware assignment.

Reply:
The proposed change is in the latest commit

Reply:
This would also work, for some reason I thought there was an explicit destructor here that was forcing us into a rule-of-3/rule-of-5 situation